packages/client/src/Conversation.tsx

```diff
@@ -239,9 +235,10 @@ function Presentable({
               className={clsx(classes.expand, {
                 [classes.expandOpen]: expanded
               })}
-              onClick={handleExpandClick}
+              onClick={() => setExpanded(!expanded)}
               aria-expanded={expanded}
               aria-label="Show more"
+              aria-controls="message-card-content"
```

Comment:
This is good - but the id needs to be unique. There are likely to be multiple presentable elements on a page.